core/new-gui/src/app/workspace/component/operator-panel/operator-label/operator-label.component.ts

```diff
@@ -16,9 +20,14 @@ import { OperatorSchema } from '../../../types/operator-schema.interface';
 })
 export class OperatorLabelComponent implements OnInit, AfterViewInit {
 
+  @ViewChild('ngbTooltip') tooltipWindow: NgbTooltip | undefined;
   @Input() operator?: OperatorSchema;
   public operatorLabelID?: string;
 
+  private mouseEnterSubject$ = new Subject<void>();
+  private mouseLeaveSubject$ = new Subject<void>();
+  private openCommandsObservable$ = new Observable<void>();
```

Comment:
I don't like the naming for these subjects and observable. Having the `$` is not really consistent with how we defined these in other modules. For instance:
`operatorAddSubject = new Subject<OperatorPredicate>();`
`executeEndedStream = new Subject<ExecutionResult>();`